### PR TITLE
fix(updatemanifest): update the version of manifest after upgrade

### PR DESCRIPTION
**Pull request by NamanJain8**
We were not updating the manifest version after the updation. This PR fixes that.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/7828)

### CODE REVIEW

File: ee/updatemanifest/run.go [hunk 1]
```diff
@@ -101,6 +102,7 @@ func run() error {
 			for _, pred := range preds {
 				ns, attr, err := parseNsAttr(pred)
 				if err != nil {
+					parsedPreds = append(parsedPreds, pred)
```
Comment (ahsanbarkati) [inline ee/updatemanifest/run.go:105]: Why is this required?

Comment (NamanJain8) [reply ee/updatemanifest/run.go:105]: As we are overwriting the manifest, I think it should be a good idea to only fix what the tool can fix. It is better to leave the erroneous predicate in the manifest so that the user can fix/remove it, rather than the tool removing it.

**Review by ahsanbarkati — APPROVED** [review]
